packages/type-storage/src/fromMetadata/getHasher.ts

```diff
@@ -2,41 +2,19 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Text } from '@polkadot/types';
 import { StorageHasher } from '@polkadot/types/primitive';
 import { blake2AsU8a, xxhashAsU8a } from '@polkadot/util-crypto';
 
-type HasherInput = string | Buffer | Uint8Array;
```

Comment:
Do we need the `string` in the type definition anywhere else than here? https://github.com/polkadot-js/api/blob/master/packages/type-storage/src/fromMetadata/createFunction.ts#L82